faqs.md

```diff
@@ -0,0 +1,155 @@
+<img src="./images/logos/colour text.png" alt="GMN logo, a coloured GMN logo with transparent background." width="400"  align="right" />
+
+# Frequently Asked Questions (FAQs)
+
+Contents
+
+[Galaxy](#Galaxy)
+
+[The Galaxy Mentor Network Program](#GMN)
+
+[The Gmentees](#Gmentees)
+
+[The Gmentors](#Gmentors)
+
+More FAQs will be added to this page as questions arise.
+
+## Galaxy <a name="Galaxy"></a>
```

Comment:
```suggestion
## Galaxy
```

Reply:
The anchor would not be needed. The template system should be able to generate them automatically